docs/editorial-workflow.md

```diff
@@ -27,16 +26,14 @@ publish_mode: editorial_workflow
 
 There are no other configuration options right now. There are always three possible statuses, and new branch names are created according to the pattern `cms/collectionName-entrySlug`.
 
-
 ## About metadata
 
 Netlify CMS embraces the idea of Git-as-backend for storing metadata. The first time it runs with the editorial_workflow setup, it creates a new ref called `meta/_netlify_cms`, pointing to an empty, orphan tree.
 
 Actual data are stored in individual `json` files committed to this tree.
 
-
 ## Implementation
 
 Instead of adding logic to `CollectionPage` and `EntryPage`, the Editorial Workflow is implemented as Higher Order Components, adding UI and dispatching additional actions.
 
-Furthermore, all editorial workflow state is managed in Redux - there's an `actions/editorialWorkflow.js` and a `reducers/editorialWorkflow.js` files.
+Furthermore, all editorial workflow states are managed in Redux - there are the `actions/editorialWorkflow.js` and the `reducers/editorialWorkflow.js` files.
```

Comment:
Makes sense. My misunderstanding of the syntax involved. I'll remove this change.